static/char_counter/js/char_counter.js

```diff
@@ -0,0 +1,52 @@
+let LineLengthSettingSingleton = 0;
```

Comment:
No idea. Do you have a preference? This is how my current team likes it. Although this is a little bit different as it's a singleton, not a const. I'm tempted to remove the singleton by wrapping these functions into a class so that can become a property of the class instead of an ugly global singleton

Reply:
I like classes